pom.xml

```diff
@@ -120,7 +120,6 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/x
     <maven-compiler-plugin.version>3.10.1</maven-compiler-plugin.version>
 
     <shell-executable>bash</shell-executable>
-
```

Comment:
cut this change; i want the most minimal backport possible

Reply:
Sure thing, this was done just to trigger the full test run (and hence we see hdfs javadoc failure because all modules were involved in the build runs)